plugins/providers/gcloudiam/provider.go

```diff
@@ -75,12 +104,19 @@ func (p *Provider) GrantAccess(pc *domain.ProviderConfig, a *domain.Appeal) erro
 	}
 
 	if a.Resource.Type == ResourceTypeProject || a.Resource.Type == ResourceTypeOrganization {
-		if err := client.GrantAccess(a.AccountType, a.AccountID, a.Role); err != nil {
-			if errors.Is(err, ErrPermissionAlreadyExists) {
-				return nil
-			}
+		requestedRole, err := p.GetRequestedRoleInAppeal(pc, a)
+		if err != nil {
 			return err
 		}
+		for _, p := range requestedRole.Permissions {
```

Comment:
after changes in https://github.com/odpf/guardian/pull/220, we shouldn't retrive permissions from `RoleConfig` anymore. Let's use `appeal.Permissions` instead

Reply:
Done